.github/workflows/integration_test_run.yaml

```diff
@@ -196,51 +188,30 @@ jobs:
           microk8s-addons: ${{ inputs.microk8s-addons }}
           channel: ${{ inputs.channel }}
           juju-channel: ${{ inputs.juju-channel }}
-      - name: Enable microk8s registry
-        if: ${{ inputs.provider == 'microk8s' && github.event.pull_request.head.repo.fork }}
-        run: |
-          sudo microk8s enable registry
-          sudo microk8s kubectl -n container-registry rollout status -w deployment/registry
       - name: Download all artifacts
         uses: actions/download-artifact@v3
-        if: ${{ github.event.pull_request.head.repo.fork }}
-      - name: Push images to microk8s registry
-        if: ${{ inputs.provider == 'microk8s' && github.event.pull_request.head.repo.fork }}
+      - name: Enable microk8s registry and push images
+        if: ${{ inputs.provider == 'microk8s' }}
         run: |
+          sudo microk8s enable registry
+          sudo microk8s kubectl -n container-registry rollout status -w deployment/registry
+          ls
           for image_name in $(echo '${{ inputs.images }}' | jq -cr '.[]'); do
-            docker load < ${image_name}/${image_name}.tar
+            cat ${image_name}/${image_name}.tar | docker import - localhost:32000/${image_name}:latest
             docker push localhost:32000/${image_name}:latest
           done
```

Comment:
I think we can directly import to microk8s without having to import and push
`microk8s ctr images import - `
:D

Reply:
Indeed. Thx!